app/src/main/kotlin/com/simplemobiletools/dialer/extensions/Activity.kt

```diff
@@ -83,7 +84,9 @@ fun SimpleActivity.getHandleToUse(intent: Intent?, phoneNumber: String, callback
                 }
                 defaultHandle != null -> callback(defaultHandle)
                 else -> {
-                    SelectSIMDialog(this, phoneNumber) { handle ->
+                    SelectSIMDialog(this, phoneNumber, onDismiss = {
+                        if(this is DialerActivity) finish()
```

Comment:
use our code style please, add the missing space and brackets

Reply:
Oh, got that.